### PR TITLE
fix: missing private property in reading history query

### PR DESCRIPTION
**Pull request by sshanzel**
### Describe what this PR does
- The reading history query was missing the private property.
- Also fixed the selections of Squad to be the active ones only.
- Let's open a ticket to consolidating our queries to use a Fragment for better maintenance I think.

## Events

Did you introduce any new tracking events?
Don't forget to update the [Analytics Taxonomy sheet](https://docs.google.com/spreadsheets/d/18Lv7zXges9QfVX5VYL1a-Hyl0e1sQ3sLr0OK8YZWKXI/edit#gid=0)

Log the new/changed events below:

| Type   | event_name  | value |
|--------|-------------|-------|
| Change/New | event name  | extra: { ... } |

### On those affected packages:
- [ ] Have you done sanity checks in the webapp?
- [ ] Have you done sanity checks in the extension?
- [ ] Does this not break anything in companion?

### Did you test the modified components media queries?
- [ ] MobileL (420px)
- [ ] Tablet (656px)
- [ ] Laptop (1020px)

#### Did you test on actual mobile devices?
- [ ] iOS (Chrome and Safari)
- [ ] Android

WT-{number} #done

### CODE REVIEW

File: packages/shared/src/graphql/users.ts [hunk 1]
```diff
@@ -181,6 +181,7 @@ const READING_HISTORY_FRAGMENT = gql`
       numUpvotes
       bookmarked
       numComments
+      private
```
Comment (rebelchris) [inline packages/shared/src/graphql/users.ts:184]: This is not great, it should still show in your actual reading history, just not the ones that are used for sharing I think.

**Review by rebelchris — CHANGES_REQUESTED** [review]

File: packages/shared/src/graphql/users.ts (hunk 1 above)
Comment (sshanzel) [reply packages/shared/src/graphql/users.ts:184]: We already have the condition in the ShareModal to not show private posts. In reading history, the context menu also have that logic.

Comment (rebelchris) [reply packages/shared/src/graphql/users.ts:184]: Sorry my bad, I was thinking you introduced "isPublic" 😅

**Review by rebelchris — APPROVED** [review]
Good to go, thanks!

**Pull request merged**